default/mapping_sheet.py

```diff
@@ -0,0 +1,37 @@
+import jsonref
+
+from io import StringIO
+from django.http.response import HttpResponse
+from libcoveocds.config import LibCoveOCDSConfig
+from libcoveocds.schema import SchemaOCDS
+from ocdskit.mapping_sheet import mapping_sheet as mapping_sheet_method
+
+
+def get_mapping_sheet_from_url(url):
+    schema = jsonref.load_uri(url)
+
+    return _get_mapping_sheet(schema)
+
+
+def get_mapping_sheet_from_uploaded_file(uploaded):
+    schema = jsonref.loads(uploaded.read())
+
+    return _get_mapping_sheet(schema)
+
+
+def get_extended_mapping_sheet(extensions):
+    cove_config = LibCoveOCDSConfig()
+    schema_ocds = SchemaOCDS('1.1', extensions, lib_cove_ocds_config=cove_config)
```

Comment:
I suggest instead using the ProfileBuilder from extension_registry.py

It has the advantage over SchemaOCDS of being [documented](https://ocdsextensionregistry.readthedocs.io/en/latest/api/profile_builder.html), having a stable API, and being used in OCDS Kit, [see example usage](https://github.com/open-contracting/ocdskit/blob/master/ocdskit/cli/commands/mapping_sheet.py#L29-L31)

Reply:
Thanks! I will update this.